app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/provision_workflow.rb

```diff
@@ -122,6 +122,12 @@ def allowed_cloud_volumes(_options = {})
     Hash[cloud_volumes || {}]
   end
 
+  def allowed_placement_groups(_options = {})
+    return {} if ar_ems.nil?
+    
```

Comment:
Make sure to remove the trailing whitespace here

Reply:
@agrare 
I will remove if you think I should, but just noting that I had originally left the white space in, even with the rubocop warning, to follow the same pattern as similar methods within this file.
https://github.com/ManageIQ/manageiq-providers-ibm_cloud/blob/01085d1c7c5dffd4d18932dd936a2f45a0b7ead0/app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/provision_workflow.rb#L57-L124

Reply:
All the other methods in this file are wrong then :smile: I'm guessing they were merged while miq-bot was down since github's diff doesn't show it

Reply:
ok :)

Reply:
Oh, I think there was a miscommunication.  The issue isn't the extra newline it was the trailing whitespace.

So:
```
def allowed_instance_type(_options = {}) 
  return {} if ar_ems.nil? 

  if sap_image?
```

Is fine, but

```
def allowed_instance_type(_options = {}) 
  return {} if ar_ems.nil? 
  
  if sap_image?
```

Is the issue